test/runtests.jl

```diff
@@ -122,6 +122,34 @@ end
     @test ShortString7(ShortString7("ab")) isa ShortString7
 
     @test_throws ErrorException ShortString3(ShortString7("123456"))
+
+    # Test creating shortstrings with maxlen
+    @test ShortString(
+        ss127"Be honest, do you actually need a string longer than this. Seriously. C'mon this is pretty long.",
+        127,
+    ) == "Be honest, do you actually need a string longer than this. Seriously. C'mon this is pretty long."
+    @test ShortString(
+        ss63"Basically a fairly long string really", 63
+    ) == "Basically a fairly long string really"
+    @test ShortString(ss31"A Longer String!!!", 31) == "A Longer String!!!"
```

Comment:
I don't think we need these test.
They don't test anything different do they?

We should test things with:
 - shortstring below maxlen
 - shortstring at maxlen
 - shortstring above maxlen
 - The above but using regular `String` and `SubString`